app/src/main/java/com/example/geoquiz/MainActivity.kt

```diff
@@ -20,26 +20,19 @@ class MainActivity : AppCompatActivity() {
     private lateinit var cheatButton: Button
     private lateinit var questionTextView: TextView
 
-    companion object {
-        private const val TAG = "MainActivity"
-        private const val KEY_CURRENT_INDEX = "index"
-        private const val REQUEST_CODE_CHEAT = 0
-    }
-
     private val quizViewModel by lazy {
         ViewModelProviders.of(this).get(QuizViewModel::class.java)
     }
```

Comment:
ViewModelProviders is deprecated and should be replaced with ViewModelProvider
E.g: `ViewModelProvider(this).get(QuizViewModel::class.java)`